django_docker_engine/docker_utils.py

```diff
@@ -1,6 +1,7 @@
 import json
 import logging
 import os
+
```

Comment:
This is probably the source of the error:
```
ERROR: /home/travis/build/refinery-platform/django_docker_engine/django_docker_engine/docker_utils.py Imports are incorrectly sorted.
```

I'll add a `grep ERROR` in test.sh so the problem will be more obvious. In my court. Thanks!

Reply:
Thanks! I was actually confused as to why this build failed: https://travis-ci.org/refinery-platform/django_docker_engine/jobs/365305011 I couldn't find anything wrong with it :(